src/Sarif/Map/JsonMapSettings.cs

```diff
@@ -30,7 +30,7 @@ public class JsonMapSettings
         /// <param name="mapMaximumSizeBytes">Map size limit, in bytes (ex: 10 * JsonMapSettings.Megabyte), 0 for no limit</param>
         public JsonMapSettings(double mapSizeRatio, double mapMaximumSizeBytes = 0)
         {
-            if (mapSizeRatio <= 0 || mapSizeRatio > 100) { throw new ArgumentOutOfRangeException("maxFileSizePercentage must be > 0 and <= 100."); }
+            if (mapSizeRatio <= 0 || mapSizeRatio > 100) { throw new ArgumentOutOfRangeException(nameof(mapSizeRatio), "mapSizeRatio must be > 0 and <= 100."); }
```

Comment:
In one case, we use nameof to get the variable name, but in the other it is hard-coded.  Perhaps store nameof(mapSizeRatio) in a variable and use string.format to add it to the front of the string?

Reply:
thanks for the suggestion. changed to interpolation :)

---
In reply to: [562078645](https://github.com/microsoft/sarif-sdk/pull/2253#discussion_r562078645) [](ancestors = 562078645)